src/main/java/io/rpg/model/actions/Action.java

```diff
@@ -4,4 +4,5 @@
  * A marker interface for action classes.
  */
 public interface Action {
+  Action VOID = new Action() {};
```

Comment:
Should other types of actions be initiated here or is that just for VOID?

Reply:
Only for VOID